SummerProject/static/src/components/Footer/style.css

```diff
@@ -0,0 +1,190 @@
+.Footer{
```

Comment:
It's a good practice to name css class names starting with lowercase letter (eg `footer` instead of `Footer`)